index.html

```diff
@@ -289,6 +289,15 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <section class="contact">
+        <h3 id="form-header"> I'm always interested in hearing about new projects, so if you'd like to chat please get in</h3>
+        <form class="form" method="POST">
+          <input type="text" name="Names"  id="name">
```

Comment:
- [ ] This input tag is supposed to have a `maxlength` attribute, so the user does not enter a value that is too long.